### PR TITLE
for issue 164 - local codacy

### LINKED ISSUE #164: Introduce local codacy flow
**Is your feature request related to a problem? Please describe.**
[This PR on developers.md](https://github.com/oceanprotocol/ocean.py/pull/145) had [codacy test failures](https://app.codacy.com/gh/oceanprotocol/ocean.py/pullRequest?prid=6996426). 

It made me realize that we don't currently have a flow to run codacy locally. Having that would allow much more rapid iterations.

**Describe the solution you'd like**

- [x] figure out how to run codacy locally
- [x] update developers.md with instructions for devs to run codacy locally

**More info**

- Using codacy locally: https://docs.codacy.com/related-tools/running-local-analysis/
- Follow-up issue: #174 "Get all codacy issues passing"

### PR DESCRIPTION
**Pull request by trentmc**
Fixes #164 "Introduce local codacy flow; fix codacy issues in developers.md"

### CODE REVIEW

File: READMEs/developers.md [hunk 1]
```diff
@@ -132,3 +132,20 @@ Specifically, [follow this workflow](https://docs.oceanprotocol.com/concepts/con
...
 Specifically, [follow the Release Process instructions](../RELEASE_PROCESS.md).
+
+
+## Bonus: Test code quality
+
+We use Codacy to test for quality of code. Here's how.
+
+Export Codacy project token. If needed, [generate your own](https://docs.codacy.com/repositories-configure/integrations/project-api/)
+```console
+export CODACY_PROJECT_TOKEN=2f4be1e422ca43b19c506ec97bebd9eb
```
Comment (calina-c) [inline READMEs/developers.md:143]: Is this the "real" value? Is it suppposed to be public? In the Github Action, we are encrypting it. I always thought it was supposed to be a secret, but I might be wrong.

**Comment by calina-c** [comment]
@trentmc Can you also fix the issue that Codacy has (with your readme.md changes *about* Codacy 😅)?

File: READMEs/developers.md (hunk 1 above)
Comment (trentmc) [reply READMEs/developers.md:143]: Yep, likely supposed to be private. That's why this PR is still in "draft" mode. By the time the final PR goes through, it won't be there. Just trying to have something to work against while I figure this out.

**Comment by trentmc** [comment]
> Can you also fix the issue that Codacy has (with your readme.md changes about Codacy sweat_smile)?

Yes, that's part of the goal of the issue. See its checklist.

**Comment by trentmc** [comment]
BTW I will be able to get back to this in about 30 min. It feels like I go to solve one issue, and two more pop up that need resolving first. But we're knocking them down:)

**Review by calina-c — APPROVED** [review]

**Pull request merged**